src/template-linter.ts

```diff
@@ -1,13 +1,21 @@
-import { Diagnostic, DiagnosticSeverity, Files, TextDocument } from 'vscode-languageserver';
+import { Diagnostic, Files, TextDocument } from 'vscode-languageserver';
 import { uriToFilePath } from 'vscode-languageserver/lib/files';
 import { hasExtension } from './utils/file-extension';
+import { toDiagnostic } from './utils/diagnostic';
 
 import * as path from 'path';
 import * as fs from 'fs';
 
 import Server from './server';
 import { Project } from './project-roots';
 
+interface TemplateLinterError {
```

Comment:
This interface seems to be a duplicate from utils/diagnostic.ts. Can't the other one be reused?